packages/experiments/src/components/Microfeedback/Microfeedback.view.tsx

```diff
@@ -0,0 +1,178 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { withSlots } from '../../Foundation';
+import { Stack } from '../../Stack';
+import { IconButton, IButtonStyles, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { IStackStyles } from 'office-ui-fabric-react/lib/Stack';
+import { Callout } from 'office-ui-fabric-react/lib/Callout';
+import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
+import { List } from 'office-ui-fabric-react/lib/List';
+
+import { IMicrofeedbackComponent, IMicrofeedbackProps } from './Microfeedback.types';
+import { getTheme, mergeStyleSets, FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { IMicrofeedbackState } from './Microfeedback.state';
+
+import { initializeIcons } from '@uifabric/icons';
+
+const theme = getTheme();
```

Comment:
This will grab the global theme, but won't grab any contextual theming.

`createComponent` and `styled` handle all of this for you though. The recommended approach is to define a styles function and `createComponent` will automatically provide you the theme, whether it's from a prop, the context, or the global theme. It's all automatic.

Reply:
Removed